contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.7.0--3.8.0.sql

```diff
@@ -68,6 +68,7324 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION sys.babelfish_conv_datetime_to_string(IN p_datatype TEXT,
```

Comment:
Why is this change included? Is it to address BABEL-4299 upgrade issue? Do we need all these changes?

Reply:
yes, this is to address BABEL-4299 upgrade issue. Yes, we need these changes as Concat was previously used in babelfish_conv_datetime_to_string and without this older definition will still exist and concat will throw an error.